content/2017-09-12_python-komunita-a-ja.md

```diff
@@ -0,0 +1,63 @@
+Title: Python komunita a já
+Date: 2017-09-12 22:02:00
+Author: Honza Javorek
+Twitter: honzajavorek
+Gravatar: mail@honzajavorek.cz
+About: [Honza](http://honzajavorek.cz) je programátor. Od roku 2011 pomáhá budovat českou komunitu kolem jazyka [Python](http://python.cz/). Pracuje v [Apiary](https://apiary.io/), kde se stará o [Dredd](https://github.com/apiaryio/dredd), Open Source nástroj na testování webových API.
+
+
+Mnoho věcí v [české Python komunitě][python.cz] vzniklo s mou pomocí. Založil jsem [Brněnské Pyvo](https://pyvo.cz/brno-pyvo/), pomáhal jsem rozjíždět [Ostravské Pyvo](https://pyvo.cz/ostrava-pyvo/) a [pražské PyLadies](http://pyladies.cz/praha_course/), byl jsem na ustavujících schůzkách [brněnských PyLadies](http://pyladies.cz/brno_course/), pražského [PyWorkingu](https://pyworking.cz/), byl jsem v organizačních týmech všech tří [PyCon CZ](https://cz.pycon.org/) konferencí, spoluorganizoval jsem první pražské [Django Girls](http://djangogirls.org/), sepisoval jsem nové stanovy naší neziskovky [Pyvec](http://pyvec.org/), a jsem původním autorem i samotné stránky [python.cz]. Když se podívám na [Zapoj se], velká část nápadů a úkolů je ode mě.
```

Comment:
- zrušit čárku před "a jsem původním"
- chybí link k Zapoj se

Reply:
link nechybí, je na konci souboru